nameko/cli/shell.py

```diff
@@ -13,10 +13,19 @@
 from nameko.standalone.events import event_dispatcher
 
 
+SHELLS = ['bpython', 'ipython']
+
+
 def init_parser(parser):
     parser.add_argument(
         '--broker', default='amqp://guest:guest@localhost',
         help='RabbitMQ broker url')
+    parser.add_argument(
+        '--interface', choices=SHELLS,
+        help='Specify an interactive interpreter interface.')
+    parser.add_argument(
```

Comment:
don't think we need a separate option for this (think django has this for historical reasons. tried digging through the source and got as far as the svn->git conversion and lost track. from what i can tell, django originally had `--plain` to stop defaulting to ipython, and only later added `--interface` to support bpython

here i guess we could just add 'plain' as another choices in `SHELLS`?